_includes/social_links/dribbble.html

```diff
@@ -0,0 +1 @@
+<a href="https://dribbble.com/{{ include.username || site.dribbble_username }}"><svg class="svg-icon" xmlns="http://www.w3.org/2000/svg"><use xlink:href="/assets/icons.svg#dribbble"></use></svg> <span class="username">{{ site.dribbble_username }}</span></a>
```

Comment:
Each of these social templates has `include.username` which for some reason is being used instead of `site.dribble_username`  when it's probably uninitialized.